### ISSUE TITLE
Deleting from Clip History does not work correctly

### ISSUE THREAD

**Issue by hoppfrosch** [aviaryan/Clipjump#105]
Opening the Clip History (Action Menu -> H) I deleted nearly all clips from the clip history, selecting the items and pressing DEL - after this the clip history only shows 3 clips remaining.

Now when I try to insert a clip via CTRL-V and cycle through the available clips by subsequently pressing V in Paste mode, all my clips are still available (much more than the 3 remaining from clip history).

See attached animated gif:![Demo](http://i.imgur.com/SNA4KJ1.gif)

Also Adding to clip history does not work correctly - at least screen refresh fails.
Having the Clip History window open and adding a clip by copying a text into the clipboard, the clip history window is not refreshed to display the newly added clip ...

(Using the latest Version from github - https://github.com/aviaryan/Clipjump/commit/1109b06ae5d30e7e10f677ad7abf3bab0c2e12d7)

**Comment by aviaryan** [comment]
> Opening the Clip History (Action Menu -> H) I deleted nearly all clips from the clip history, selecting the items and pressing DEL - after this the clip history only shows 3 clips remaining.
Now when I try to insert a clip via CTRL-V and cycle through the available clips by subsequently pressing V in Paste mode, all my clips are still available (much more than the 3 remaining from clip history).

From the [FAQ](http://clipjump.sourceforge.net/docs/faq.html)

> The Clear History button in History Tool doesn't delete the recent Clips in Ctrl+V Paste mode?
OR
I delete a clip from Ctrl+V Paste mode but it is still there in History Tool ?
The History Tool and the Paste mode are two different components and there clips are independent of each other. 
History Tool is meant for history-keeping whereas Ctrl+V Paste mode is meant to show clips contained in different channels of Clipjump. The record of all clips added to any of the channels are shown in a single window i.e.History Tool. On the other hand, clips stored in channels can be managed from the Channel Organizer.

In other words, to delete those 970 clips clear them from Channel Organizer.

---

> Also Adding to clip history does not work correctly - at least screen refresh fails.
Having the Clip History window open and adding a clip by copying a text into the clipboard, the clip history window is not refreshed to add the newly added clip ...

Thank you for reporting.

**Comment by hoppfrosch** [comment]
I've read this earlier - but didn't remember this. :-) (Thanks for the hint) ... or I didn't understoodnd it correctly. 

I got it now: This means, I can delete all clips via Channel Organizer - and still have clips available within the Clip History (if I have'nt deleted clips from history). 

I was misleaded by the wording "History", maybe the better word would have been "Archive" (but I'm not an native english speaker ;-))

**Comment by hoppfrosch** [comment]
Discussion/brainstorming is continued within #110.
Open Bug is discussed in #106
